tests/profile/tracking/test_tracking_session.py

```diff
@@ -0,0 +1,45 @@
+import os
+import pytest
+
+from kano_profile.paths import tracker_dir
+from kano_profile.tracker.tracking_session import TrackingSession
+
+
+TEST_DATA = [
```

Comment:
This could be abstracted to a fixture that gives you a `TrackingSession` object with some data (general note).

Reply:
Nice idea. I'll give it a shot